client/src/components/Discover.tsx

```diff
@@ -151,8 +150,10 @@ const Discover:FC<any> = ({user}) => {
               </iframe>
               </Card>
 
-              <Grid item xs='auto'>
-                <Typography variant='h4' align='center'>
+              <Grid item xs='auto'
+                style={{backgroundColor: '#A4A4A4'}}
```

Comment:
One of the things they don't want us to do is hardcode color codes. If you use something like `background: theme.palette.background.paper`, the css properties can change according to the theme switcher. Take a look at how I'm doing that in profile.styles.ts and Profile.tsx. It's not too bad to implement, and you can probably reuse the profile.styles.ts in this.